src/com/sap/piper/integration/TransportManagementService.groovy

```diff
@@ -44,6 +44,12 @@ class TransportManagementService implements Serializable {
             ]
         ]
 
+        def proxy = config.proxy ? config.proxy : script.env.HTTP_PROXY
```

Comment:
We have three times the same strategy for resolving the proxy. I would like to suggest to move that to a method in order to avoid having the same strategy three times (reduces maintenance efforts).